### PR TITLE
fix(theme): add new color codes

### PR DESCRIPTION
**Pull request by bansalayush**
adding rebranding colors

### CODE REVIEW

**Comment by kamleshchandnani** [comment]
Deploy preview for *leaf-ui* processing.

Built with commit 3a87fc17ba8b7abf05a1c150a14cf06e15ea30ce

https://app.netlify.com/sites/leaf-ui/deploys/5bd90250c6aed648b30e4454

File: src/theme/web/theme.js [hunk 1]
```diff
@@ -37,6 +37,13 @@ theme.color.greyLight = '#dbdbdb';
 theme.color.grey = '#aeaeae';
 theme.color.greyDark = '#727171';
 theme.color.greyDarker = '#4a4a4a';
+theme.color.jade = '#03B778'; // primary
+theme.color.jadeDark = '#108B65';
+theme.color.jadeLight = '#7ACAAE';
+theme.color.jadeLighter = '#E3F4DE';
+theme.color.lime = '#71C95B'; // trend
+theme.color.corn = '#FFD067'; // trip;
+theme.color.purple = '#BEABE9'; // tryst
```
Comment (kashishgrover) [inline src/theme/web/theme.js]: Remove the comments. As per design, you can always map the colors with the correct hex code.

File: src/theme/web/theme.js [hunk 2]
```diff
@@ -37,6 +37,13 @@ theme.color.greyLight = '#dbdbdb';
 theme.color.grey = '#aeaeae';
 theme.color.greyDark = '#727171';
 theme.color.greyDarker = '#4a4a4a';
+theme.color.jade = '#03B778';
```
Comment (kamleshchandnani) [inline src/theme/web/theme.js]:
```suggestion
theme.color.jadeLighter = '#E3F4DE';
theme.color.jadeLight = '#7ACAAE';
theme.color.jade = '#03B778';
theme.color.jadeDark = '#108B65';
```

File: src/theme/web/theme.js [hunk 3]
```diff
@@ -37,6 +37,13 @@ theme.color.greyLight = '#dbdbdb';
...
+theme.color.jadeDark = '#108B65';
```
Comment (kamleshchandnani) [inline src/theme/web/theme.js]: Also what about `jadeDarker`?

File: src/theme/web/theme.js (hunk 2 above)
Comment (kamleshchandnani) [reply src/theme/web/theme.js]: This should be the order. So it's readable as well

**Comment by kamleshchandnani** [comment]
Also just confirm once with the design team, if jade is the new primary color then the changes mentioned by @kashishgrover should also be incorporated.

**Comment by bansalayush** [comment]
@kamleshchandnani  if we make jade as primary color then there would be changes in CRS too . Design team hasn't replied back yet about color changes in CRS.

**Comment by bansalayush** [comment]
@kamleshchandnani  ok

File: src/theme/web/theme.js (hunk 3 above)
Comment (bansalayush) [reply src/theme/web/theme.js]: According to design `darker` shade is not being used . Also making jade as primary color will be a leaf-ui refactor when all the components are designed according to rebranding

**Pull request merged**